db/20150705131449_add_type_to_reports.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToReports < ActiveRecord::Migration
```

Comment:
Reappeared in rebase? Needs to be removed. (we already have https://github.com/theforeman/foreman/blob/develop/db/migrate/20150705131449_add_type_to_reports.rb)